### PR TITLE
Part 4 - fix: spec lacking conformance section are informative

### LINKED ISSUE #2051: If a spec lacks <section id=conformance>, then it reference should be informative
## Important info

* URL to affected spec:
https://w3c.github.io/dxwg/ucr/

## Description of problem
Specs can have either Informative or Normative references. By default, ReSpec treats all documents as if they were Normative. However, some documents, like Use Cases docs, generally are don't use Normative references.  

But what I was expecting?:
The citations in a document that doesn't declare a conformance section (i.e., `<section id="conformance">`) to be treated as Informative. 

To achieve this, all we need to do is add "informative" to the class `attribute` of the `<body>` element  here: 
https://github.com/w3c/respec/blob/develop/src/w3c/conformance.js#L10

cc @andrea-perego

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #2051

### CODE REVIEW

File: src/core/inlines.js [hunk 1]
```diff
@@ -100,25 +106,21 @@ export function run(conf) {
...
-            ref = ref.replace(/^(!|\?)/, "");
-            df.appendChild(document.createTextNode("["));
-            const refHref = `#bib-${ref.toLowerCase()}`;
-            const cite = hyperHTML`<cite><a class="bibref" href="${refHref}">${ref}</a></cite>`;
-            df.appendChild(cite);
-            df.appendChild(document.createTextNode("]"));
-
-            if (illegal && !conf.normativeReferences.has(ref)) {
+            const cite = renderInlineCitation(ref);
+            const cleanRef = ref.replace(/^(!|\?)/, "");
```
Comment (saschanaz) [inline src/core/inlines.js]: Previously the code removed `!`/`?` first and this now doesn't. Would there be a behavior change?

Comment (marcoscaceres) [reply src/core/inlines.js]: Well spotted. It's now done in by `renderInlineCitation()` - so same behavior.

File: tests/spec/core/idl-index-spec.js [hunk 2]
```diff
@@ -73,10 +74,10 @@ interface Bar {
       body,
     };
     const doc = await makeRSDoc(ops);
-    const idlIndex = doc.getElementById("idl-index");
-    expect(idlIndex.innerText.includes("Foo")).toEqual(false);
-    expect(idlIndex.innerText.includes("Bar")).toEqual(true);
-    expect(idlIndex.innerText.includes("Baz")).toEqual(false);
+    const { textContent } = doc.querySelector("#idl-index pre");
+    expect(textContent).toMatch(/Include/gm);
```
Comment (saschanaz) [inline tests/spec/core/idl-index-spec.js]: Maybe we can use `toContain()` here?

File: tests/spec/core/idl-index-spec.js [hunk 3]
```diff
@@ -85,44 +86,45 @@ interface Bar {
...
       config: makeBasicConfig(),
       body,
     };
     const doc = await makeRSDoc(ops);
     const idlIndex = doc.getElementById("idl-index");
-    expect(idlIndex.innerText.includes("Foo")).toEqual(false);
-    expect(idlIndex.innerText.includes("Bar")).toEqual(false);
-    expect(idlIndex.innerText.includes("Baz")).toEqual(false);
-    expect(idlIndex.innerText.includes("Qux")).toEqual(true);
+    expect(idlIndex.textContent.includes("Example")).toEqual(false);
```
Comment (saschanaz) [inline tests/spec/core/idl-index-spec.js]: Also here.

File: tests/spec/core/idl-index-spec.js [hunk 4]
```diff
@@ -131,36 +133,37 @@ interface Bar {
...
     const ops = {
       config: makeBasicConfig(),
       body,
     };
     const doc = await makeRSDoc(ops);
     const idlIndex = doc.getElementById("idl-index");
-    expect(idlIndex.innerText.includes("Foo")).toEqual(true);
-    expect(idlIndex.innerText.includes("Bar")).toEqual(false);
-    expect(idlIndex.innerText.includes("Baz")).toEqual(false);
+    expect(idlIndex.textContent.includes("Pass")).toBe(true);
```
Comment (saschanaz) [inline tests/spec/core/idl-index-spec.js]: Here too

File: tests/spec/core/idl-index-spec.js [hunk 5]
```diff
@@ -264,6 +267,7 @@ dictionary PromptResponseObject {
         <h2>PASS</h2>
         <p>Custom paragraph.</p>
       </section>
+      <section id="conformance"></section>
```
Comment (saschanaz) [inline tests/spec/core/idl-index-spec.js]: Multiple elements with the same ID are a weird situation, what's this for?

**Comment by marcoscaceres** [comment]
Codacy and ESLint disagree 😂

**Comment by marcoscaceres** [comment]
Removed the Codacy webhook. It's was nice, but we don't need it anymore.

File: tests/spec/core/inlines-spec.js [hunk 6]
```diff
@@ -19,23 +20,26 @@ describe("Core - Inlines", () => {
     const doc = await makeRSDoc(ops);
 
     const norm = [...doc.querySelectorAll("#normative-references dt")];
-    expect(norm.length).toEqual(3);
+    expect(norm.length).toEqual(4);
```
Comment (saschanaz) [inline tests/spec/core/inlines-spec.js]: Also `toBe`?

File: tests/spec/core/idl-index-spec.js (hunk 5 above)
Comment (marcoscaceres) [reply tests/spec/core/idl-index-spec.js]: this is ok I think... It's part of a different test, but above it looks like the same test.

Comment (marcoscaceres) [reply tests/spec/core/idl-index-spec.js]: (note the above is on line 203, and this is line 268)

Comment (saschanaz) [reply tests/spec/core/idl-index-spec.js]: Oops, my mistake, sorry.

**Comment by marcoscaceres** [comment]
Thanks for the speedy review @saschanaz!

**Pull request merged**